packages/ui/src/design-system/Badge/Badge.tsx

```diff
@@ -21,7 +21,7 @@ export type BadgeProps = {
   /** Id pour les tests e2e */
   dataTest?: string;
   /** Libellé affiché dans le badge */
-  title: React.ReactNode;
+  title?: React.ReactNode;
```

Comment:
Hello, je checkais ce qu'il y avait en modif sur le DS.

Ce serait cool de throw une erreur si ni le titre, ni l'icon n'est défini sinon on peut se retrouver avec un badge vide qui n'apporte rien.

Exemple: 
`if (!title && !icon) {
    throw new Error('At least one of "title" or "icon" has to be defined');
  }`